util/compile-btc.sh

```diff
@@ -6,5 +6,5 @@ function compile_btc {
 
   ./autogen.sh
   ./configure
-  make -j5
+  make -j"$(($(nproc)+1))"
```

Comment:
This needs to be made system dependent as there's no `nproc` on macOS. We would have to use `sysctl -n hw.logicalcpu` in case of mac
```suggestion
# Mac
if [ $SYS_TYPE = "MAC" ]; then 
  make -j"$(($(sysctl -n hw.logicalcpu)+1))"
# Linux
else
  make -j"$(($(nproc)+1))"
fi
```